src/Server/Middleware/TransactionMarkerMiddlewareHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @author    Craig Gosman <craig@ingenerator.com>
+ * @licence   BSD-3-Clause
+ */
+
+namespace Ingenerator\CloudTasksWrapper\Server\Middleware;
+
+use DateTimeImmutable;
+use Ingenerator\PHPUtils\DateTime\DateString;
+
+class TransactionMarkerMiddlewareHelper
+{
+    public static function addTransactionMarkerHeaders(array $task, string $uuid, DateTimeImmutable $expiry): array
+    {
+        $task['options']['headers'][TransactionMarkerMiddleware::TRANSACTION_MARKER_HEADER] = $uuid;
```

Comment:
Possibly better to just pass through / return the `options` part of the array (which has a structure broadly defined by the wrapper) than the next level up (which might not be present depending on the structure of the code that creates the task)? Just avoids any potential for confusion.

e.g. you can always do:
```
$options = TransactionMarkerMiddlewareHelper::addTMH($options, ...);
// or
$stuff['options'] = TransactionMarkerMiddlewareHelper::addTMH($stuff['options'], ...);
```

But if you so far have the options as a top-level array it'd be a lot messier to do this:

```
$options = TransactionMarkerMiddlewareHelper::addTMH(['options' => $options], ...)['options'];
```